cluster/examples/kubernetes/monitoring/service-monitor.yaml

```diff
@@ -15,5 +15,5 @@ spec:
       rook_cluster: rook
   endpoints:
   - port: http-metrics
-    path: /
+    path: /metrics
```

Comment:
what changed to make the metrics only available on `/metrics` now?

Reply:
Maybe a Ceph Prometheus Metrics Module update, that may have gotten in due to packages used now instead of our "own" built version.